lib/logstash/filters/translate.rb

```diff
@@ -193,4 +177,58 @@ def filter(event)
       @logger.error("Something went wrong when attempting to translate from dictionary", :exception => e, :field => @field, :event => event)
     end
   end # def filter
+
+  private
+
+  def load_dictionary(registering=false)
```

Comment:
Perhaps this variable could be named `raise_exceptions` rather than `registering` in this function. That's probably more semantically clear.